agave_app/AppearanceSettingsWidget.cpp

```diff
@@ -369,6 +384,34 @@ QAppearanceSettingsWidget::createSkyLightingControls()
   return section;
 }
 
+void
+QAppearanceSettingsWidget::OnFlipX(bool value)

```

Comment:
just making a note of my suggestion to change these 3 functions to one that takes in an `axis` value to reduce the copied code